src/main/app/case/answers/getAnswerRows.ts

```diff
@@ -110,15 +110,72 @@ export const getAnswerRows = function (
           continue;
         }
         const customAnswerWithHtml = this.ctx.stepAnswersWithHTML?.[step.url]?.[fieldKey];
+        const stepLinks = isCompleteCase ? undefined : this.ctx.stepLinks[step.url];
```

Comment:
is this needed? because showActions is set to false, so the links shouldnt appear.

Reply:
Yeah when I tried using just this.ctx.stepLinks[step.url]; it would error out